packages/application-generic/src/usecases/get-decrypted-integrations/get-decrypted-integrations.usecase.ts

```diff
@@ -18,8 +22,8 @@ export class GetDecryptedIntegrations {
   async execute(
     command: GetDecryptedIntegrationsCommand
   ): Promise<IntegrationEntity[]> {
-    const query: Partial<IntegrationEntity> & { _environmentId: string } = {
-      _environmentId: command.environmentId,
+    const query: Partial<IntegrationEntity> & { _organizationId: string } = {
```

Comment:
Actually this is the task definition of done. Funnily to be able to do this we needed to do a ton of things before.